### PR TITLE
Replace references to "traces" with "distributed request".

### LINKED ISSUE #28: Do not refer to "traces" in the baggage spec
Text keeps referring to "trace", which is only tangentially related. It would be useful to introduce a term of "distributed request" or "distributed execution" at the beginning and use that, instead of "trace".

_Originally posted by @yurishkuro in https://github.com/w3c/baggage/pull/27_

### PR DESCRIPTION
**Pull request by kalyanaj**
Replace references to "traces" with "distributed request". Resolves #28.

***
<a href="https://pr-preview.s3.amazonaws.com/kalyanaj/baggage/pull/71.html" title="Last updated on Jul 28, 2021, 2:43 AM UTC (a042564)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/baggage/71/b9317d7...kalyanaj:a042564.html" title="Last updated on Jul 28, 2021, 2:43 AM UTC (a042564)">Diff</a>

### CODE REVIEW

File: abstract/common-abstract.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Distributed tracing is a set of tools and practices to monitor the health and reliability of a distributed application. A distributed application is an application that consists of multiple components that are deployed and operated separately. It is also known as micro-service.
```
Comment (yurishkuro) [inline abstract/common-abstract.md:1]: It seems most of this content is not relevant to baggage, there's really no need to mention distributed tracing.

File: baggage/HTTP_HEADER_FORMAT.md [hunk 2]
```diff
@@ -113,7 +113,7 @@ For example, if all of your data needs to be sent to a single node, you could pr
 baggage: serverNode=DF:28
 ```
 
-For example, if you need to log the original user ID when making transactions arbitrarily deep into a trace.
+For example, if you need to log the original user ID when making transactions arbitrarily deep into a distributed request.
```
Comment (yurishkuro) [inline baggage/HTTP_HEADER_FORMAT.md:116]: Sentence doesn't really work. The overall example is about annotating logs with request-specific information.

File: abstract/common-abstract.md (hunk 1 above)
Comment (dyladan) [reply abstract/common-abstract.md:1]: I agree with Yuri. Baggage could be used by any distrubuted system including one that is not traced at all.

File: abstract/common-abstract.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-Distributed tracing is a set of tools and practices to monitor the health and reliability of a distributed application. A distributed application is an application that consists of multiple components that are deployed and operated separately. It is also known as micro-service.
+Distributed tracing is a set of tools and practices to monitor the health and reliability of a distributed application. A distributed application is an application that consists of multiple components or microservices that are deployed and operated independently.
```
Comment (dyladan) [inline abstract/common-abstract.md:1]:
```suggestion
A distributed application is an application that consists of multiple components, sometimes called microservices, that are deployed and operated independently.

```

File: baggage/HTTP_HEADER_FORMAT.md (hunk 2 above)
Comment (dyladan) [reply baggage/HTTP_HEADER_FORMAT.md:116]: User-id could be considered request-specific information

File: abstract/common-abstract.md (hunk 1 above)
Comment (kalyanaj) [reply abstract/common-abstract.md:1]: Thanks, I have removed this content related to distributed tracing, and made the abstract shorter per the discussion in the last WG meeting.

File: baggage/HTTP_HEADER_FORMAT.md (hunk 2 above)
Comment (kalyanaj) [reply baggage/HTTP_HEADER_FORMAT.md]: Thanks, I have reworded this statement now to attempt to improve this.

**Review by yurishkuro — APPROVED** [review]

**Review by dyladan — APPROVED** [review]

**Review by SergeyKanzhelev — APPROVED** [review]

**Pull request merged**